conf/tikv.yml

```diff
@@ -46,7 +46,15 @@ server:
   # maximum number of messages can be processed in one tick.
   # messages-per-tick: 4096
 
-  # compression type for grpc channel, available values are none, deflate and gzip.
+  # Status address.
+  # This is used for reporting the status of TiKV directly through the HTTP address.
+  # Empty string means disabling it.
+  # status-addr: "127.0.0.1:20180"
```

Comment:
User shouldn't config status-addr in tikv.yml, see https://github.com/pingcap/tidb-ansible/blob/master/roles/tikv/templates/run_tikv_binary.sh.j2#L36.